### PR TITLE
Add config file for Travis

### PR DESCRIPTION
**Pull request by abdullahselek**
With this commit we should able to build app on Travis-CI

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  - xcodebuild -version
+  - xcodebuild -showsdks
+
+# Build App in Debug and Run Tests if specified
+  - if [ $RUN_TESTS == "YES" ]; then
+      travis_retry xcodebuild -project "$PROJECT" -sdk "$SDK" -destination "$DESTINATION" -configuration Debug ONLY_ACTIVE_ARCH=NO ENABLE_TESTABILITY=YES test;
+    else
+      travis_retry xcodebuild -project "$PROJECT" -sdk "$SDK" -destination "$DESTINATION" -configuration Debug ONLY_ACTIVE_ARCH=NO build;
+    fi
+
```
Comment (sschuberth) [inline .travis.yml]: Final newline still missing ;-)

Comment (abdullahselek) [reply .travis.yml]: I added a new line after `fi` and line numbers count is currently 36 which was 35 or I missed something else?

Comment (sschuberth) [reply .travis.yml]: I was just seeing GitHub's red icon saying "No newline at end of file", but the real issue is that while you have a newline after `fi`, you have trailing whitespace in line 36 (and line 25, BTW). Mind stripping these?

Comment (abdullahselek) [reply .travis.yml]: Just commit with stripped trailing whitespaces

**Review by sschuberth — APPROVED** [review]

**Pull request merged**